internal/cmd/api-key/command_delete.go

```diff
@@ -21,6 +21,7 @@ func (c *command) newDeleteCommand() *cobra.Command {
 	}
 
 	pcmd.AddForceFlag(cmd)
+	pcmd.AddWarnFlag(cmd)
```

Comment:
Not a huge fan of `--warn`... We have `--if-exists` elsewhere in the CLI, so that might make more sense? i.e. `confluent api-key delete A B C --if-exists`

Reply:
Maybe `--skip-invalid`?

Reply:
Also, what's the argument for adding this flag in the first place? Do we really need it? Why can't the user delete the invalid entries?